From 601e9b5f8c147ef4242b4bda76e9a16e3d635370 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Tue, 9 Jun 2015 23:30:50 -0700
Subject: [PATCH 3/8] XQuartz: Silence -Wunused-variable

X11Controller.m:939:9: warning: unused variable 'remain' [-Wunused-variable,Unused Entity Issue]
    int remain;
        ^

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit 9fe7f5ccada37e2d2a2fa92064b92a0334a3fcdd)
---
 hw/xquartz/X11Controller.m | 1 -
 1 file changed, 1 deletion(-)

diff --git a/hw/xquartz/X11Controller.m b/hw/xquartz/X11Controller.m
index c3c6976..a5c5138 100644
--- a/hw/xquartz/X11Controller.m
+++ b/hw/xquartz/X11Controller.m
@@ -936,7 +936,6 @@ extern char *bundle_id_prefix;
 
 - (void) applicationWillTerminate:(NSNotification *)aNotification _X_NORETURN
 {
-    int remain;
     [X11App prefs_synchronize];
 
     /* shutdown the X server, it will exit () for us. */
-- 
2.6.1

